docs/content/porter-or-duffle.md

```diff
@@ -137,7 +174,7 @@ The `porter build` command handles:
 So it's still there, but you don't have to mess with it. 😎 A few of the sections in the Porter manifest to map 1:1
 to sections in the bundle.json file, such as the bundle metadata, Parameters, and Credentials.
 
-## Hot Wiring a Bundle
+### Hot Wiring a Bundle
 I lied, they aren't actually 1:1 mappings. Porter has some special sauce, the `source` reference that makes it much
```

Comment:
For a follow-up; `source` jumped out at me -- are there lingering docs that need updating per Porter's switch from `source` value interpolation to the current mustache templating approach?

Reply:
Yeah probably. I'm doing a doc scrub before DockerCon too.

Reply:
Oops, yeah this should get updated in a follow-up. Thanks!